src/Bicep.LangServer/Handlers/BicepBuildCommandHandler.cs

```diff
@@ -83,6 +83,8 @@ private string GenerateCompiledFileAndReturnBuildOutputMessage(string bicepFileP
 
             var fileUri = documentUri.ToUri();
 
+            // Bicep file could contain load functions like loadTextContent(..). We'll refresh compilation to detect changes in files referenced in load functions.
+            compilationManager.RefreshCompilation(fileUri);
```

Comment:
@majastrz, @StephenWeatherford , this change is for build command. We have same issue for build as well.